capstone/fitness/src/test/java/com/google/sps/DataHandlerTest.java

```diff
@@ -44,28 +44,50 @@ public void testGetData() throws Exception {
 
     DataHandler dh = new DataHandler();
 
-    Entity e = new Entity("user", "hopethisisnottaken@gmail.com");
-    e.setProperty(dh.NAME_PROPERTY, "John");
-    e.setProperty(dh.AGE_PROPERTY, 18);
-    e.setProperty(dh.WEEKS_TO_TRAIN_PROPERTY, 5);
-    e.setProperty(dh.MARATHON_LENGTH_PROPERTY, 5.0);
-    e.setProperty(dh.INITIAL_TIME_PROPERTY, 2.0);
-    e.setProperty(dh.GOAL_TIME_PROPERTY, 1.0);
-    e.setProperty(dh.PROGRESS_PROPERTY, "[]");
-    e.setProperty(dh.MILE_TIME_PROPERTY, 0.2);
+    Entity user = new Entity(dh.USER_ENTITY, "hopethisisnottaken@gmail.com");
+    Entity workout = new Entity(dh.WORKOUT_ENTITY, "running");
```

Comment:
Consider pulling every value that is used in setup and also for asserting below into a variable.